express-zod-api/src/result-handler.ts

```diff
@@ -112,15 +112,17 @@ export const defaultResultHandler = new ResultHandler({
     }
     return responseSchema;
   },
-  negative: z
-    .object({
+  negative: () => {
```

Comment:
This is static, but it makes a new instance in the registry each time. I think it could cause memory leak.

Reply:
fixed by c50e58f3